recipes/fABBA/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "fABBA" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bf0672c2d0be449627380f51a7e695ddd43bf548a292de9dcfe8562e256ff95a
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+    
+requirements:
+  build:
+    - python >=3.6
+    - pip
+    - numpy 
+    - cython
+
+  run:
+    - python >=3.6
+    - pandas
+    - requests
+    - numpy
+    - tqdm
+    - scikit-learn
+    - scipy ==1.7.2
+    - matplotlib-base
+    
+
+test:
+  imports:
+    - fABBA
+
+
+about:
+  home: https://github.com/nla-group/fABBA
+  summary: 'An efficient aggregation method for the symbolic representation of temporal data.'
+  license: BSD-3-Clause
+  license_file: LICENSE
+  dev_url: https://github.com/nla-group/fABBA
+
+extra:
+  recipe-maintainers:
+    - chenxinye
+    - guettel
```

Comment:
I believe conda-forge needs @guettel to confirm that they are willing to be listed as a maintainer here